app/views/shared/_personal_key.html.slim

```diff
@@ -7,30 +7,32 @@ p.mt-tiny.mb4
 .full-width-box
   = render 'partials/personal_key/key', code: code
 
-.pkey-info.mt4.mb2
-  .sm-col.sm-col-6.sm-pr1.mb1
-    .pkey-info-image.mb1
-      = image_tag(asset_url('personal-key/warning.svg'), size: '60')
-    p = t('instructions.personal_key.once_html')
-  .sm-col.sm-col-6.sm-pl1.mb1
-    .pkey-info-image.mb1
-      = image_tag(asset_url('personal-key/lock-check.svg'), size: '60')
-    p = t('instructions.personal_key.safety_html')
-.clearfix
-.bg-lighter-blue.clearfix.mb2.px3.py2.rounded-lg
+br
```

Comment:
Wouldn't it be a good idea to convert this to HTML while we are here?

Reply:
Done.